### PR TITLE
lot refactor and rename game to app.

### PR DESCRIPTION
**Pull request by Zangetsu38**
- rename all named game to app
- Improve render on live area

# Result : 
- Get refactor bouton search/manual, using box with color like close on psvita
- Show selection on all icon clickable, ignore psn link
image

- on website link
image

### CODE REVIEW

File: vita3k/gui/src/app_context_menu.cpp [hunk 1]
```diff
@@ -104,36 +104,34 @@ static bool get_update_history(GuiState &gui, HostState &host) {
...
 
-static std::map<std::string, std::string> game_info;
+static std::map<std::string, std::string> app_info;
 
-static bool get_game_info(GuiState &gui, HostState &host) {
-    game_info.clear();
-    const auto game_path{ fs::path(host.pref_path) / "ux0/app" / host.io.title_id };
+static bool get_app_info(GuiState &gui, HostState &host) {
+    app_info.clear();
+    const auto app_path{ fs::path(host.pref_path) / "ux0/app" / host.io.title_id };
```
Comment (ichisadashioko) [inline vita3k/gui/src/app_context_menu.cpp:111]: What is the name of this weird syntax / convention used in `C/C++`?

File: vita3k/gui/src/app_context_menu.cpp [hunk 2]
```diff
@@ -104,36 +104,34 @@ static bool get_update_history(GuiState &gui, HostState &host) {
...
-        size_t game_size = 0;
-        fs::recursive_directory_iterator end;
-        for (fs::recursive_directory_iterator g(game_path); g != end; ++g) {
-            const fs::path file = *g;
-            if (fs::is_regular_file(file))
-                game_size += fs::file_size(file);
-        }
-        game_info["size"] = std::to_string(game_size / 1048576);
+        size_t app_size = 0;
+        for (const auto &app : fs::recursive_directory_iterator(app_path))
```
Comment (ichisadashioko) [inline vita3k/gui/src/app_context_menu.cpp:121]: Should we add braces `{}` to increase readability?

File: vita3k/gui/src/app_context_menu.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/app_context_menu.cpp:111]: no have understund your ask ?

Comment (ichisadashioko) [reply vita3k/gui/src/app_context_menu.cpp:111]: I'm not familiar with `C/C++` so what does this syntax means `{ var0 / var1 / var2 }`? Path concatenation? `{}` means array right? `/` is divide operator right?

Comment (Zangetsu38) [reply vita3k/gui/src/app_context_menu.cpp:111]: i have just used code inside io and other part, is same

File: vita3k/gui/src/app_context_menu.cpp (hunk 2 above)
Comment (Zangetsu38) [reply vita3k/gui/src/app_context_menu.cpp:121]: i already clear like that ^^

Comment (1whatleytay) [reply vita3k/gui/src/app_context_menu.cpp:121]: I like `{}`.

File: vita3k/gui/src/app_context_menu.cpp [hunk 3]
```diff
@@ -103,36 +103,35 @@ static bool get_update_history(GuiState &gui, HostState &host) {
...
-            const fs::path file = *g;
-            if (fs::is_regular_file(file))
-                game_size += fs::file_size(file);
+        size_t app_size = 0;
+        for (const auto &app : fs::recursive_directory_iterator(app_path)) {
+            if (fs::is_regular_file(app.path()))
+                app_size += fs::file_size(app.path());
         }
-        game_info["size"] = std::to_string(game_size / 1048576);
+        app_info["size"] = std::to_string(app_size / 1048576);
```
Comment (1whatleytay) [inline vita3k/gui/src/app_context_menu.cpp:124]: Hey zang while you're at it.
```suggestion
        app_info["size"] = std::to_string(app_size / MB(1));
```

Comment (Zangetsu38) [reply vita3k/gui/src/app_context_menu.cpp:124]: AH lol

**Pull request merged**